src/main/java/com/adobe/cq/commerce/graphql/client/HttpMethod.java

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ *
+ *    Copyright 2019 Adobe. All rights reserved.
+ *    This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License. You may obtain a copy
+ *    of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software distributed under
+ *    the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *    OF ANY KIND, either express or implied. See the License for the specific language
+ *    governing permissions and limitations under the License.
+ *
+ ******************************************************************************/
+
+package com.adobe.cq.commerce.graphql.client;
+
+/**
+ * Represents the allowed and supported HTTP methods to send GraphQL requests.
+ */
+public enum HttpMethod {
```

Comment:
Is that is actually needed? `org.apache.http.client.methods` defines constant for all http methods already, can't we use these, we have dependencies here anyway.

Reply:
That doesn't work, we need a simple `enum`, and I also wanted to make sure one cannot define anything else than `GET` or `POST`.

Reply:
Ok, make sense.